solidity/dashboard/src/components/TokenGrantVestingSchedule.jsx

```diff
@@ -1,62 +0,0 @@
-import React from 'react'
```

Comment:
It is unclear to me why do we remove this file. Is there any *unlocking* replacement or do we no longer need it?

Reply:
This component is no longer needed.